docs/tutorials/best-practices/optimisation.md

```diff
@@ -29,7 +29,7 @@ An observer is used to see how often this results in other code being run.
 
 	```Lua
 	local number = scope:Value(1)
-	local isEven = scope:Computed(function(use)
+	local isEven = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
	local isEven = scope:Computed(function(use)
```

Reply:
reverted